_episodes/03-skimage-images.md

```diff
@@ -226,60 +226,62 @@ this case, the `.tif` extension causes the image to be saved as a TIFF.
 {: .callout}
 
 
-> ## Resizing an image (20 min)
+> ## Resizing an image (10 min)
 >
-> Using your mobile phone, tablet, web cam, or digital camera, take an image.
-> Copy the image to the **Desktop/workshops/image-processing/03-skimage-images**
-> directory. Write a Python program to read your image into a variable named
-> `image`. Then, resize the image by a factor of 50 percent, using this line of
-> code:
+> Add `import skimage.transform` to your list of imports.
+> Using chair.jpg image located in the data folder, write a Python script to
+> read your image into a variable named `image`. Then, resize the image to 10 percent
+> of its current size using this line of code:
 >
 > ~~~
-> new_shape = (image.shape[0] // 2, image.shape[1] // 2, image.shape[2])
+> new_shape = (image.shape[0] // 10, image.shape[1] // 10, image.shape[2])
 > small = skimage.transform.resize(image=image, output_shape=new_shape)
 > ~~~
 > {: .language-python}
 >
 > As it is used here, the parameters to the `skimage.transform.resize()` function are the
 > image to transform, `image`, the dimensions we want the new image to have.
 >
-> Finally, write the resized image out to a new file named **resized.jpg**.
-> Once you have executed your program, examine the image properties of the
-> output image and verify it has been resized properly.
+> Next, write the resized image out to a new file named **resized.jpg** in your data directory.
+> Finally, use plt.imshow() with each of your image variables to display both images in your notebook. 
+> Don't forget to use `fig, ax = plt.subplots()` so you don't overwrite the first image with the second.
+> Images may appear the same size in jupyter, but you can see the size difference by comparing the x and y scales for each.
+> You can also see the differnce in file storage size on disk by hovering your mouse cursor over the original
+> and the new file in the jupyter file browser, using `ls -l` in your shell, or the OS file browser if it
+> is configured to show file sizes.
 >
 > > ## Solution
 > >
-> > Here is what your Python program might look like.
+> > Here is what your Python script might look like.
 > >
 > > ~~~
 > > """
-> >  * Python program to read an image, resize it, and save it
+> >  * Python script to read an image, resize it, and save it
 > >  * under a different name.
 > > """
 > > import skimage.io
 > > import skimage.transform
 > >
 > > # read in image
-> > image = skimage.io.imread(fname="chicago.jpg")
+> > image = skimage.io.imread(fname="chair.jpg")
```

Comment:
```suggestion
> > image = skimage.io.imread(fname="data/chair.jpg")
```

Reply:
Let's leave this one for now. As part of the PR to reorganise all our data files, I will update any paths used throughout the lesson.